doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -234,6 +234,8 @@ nRF9160 samples
   * Updated:
 
     * The sample now uses the new LwM2M location assistance objects through the :ref:`lib_lwm2m_location_assistance` library.
+    * Removed all read callbacks from sensor code. There is issue that using read callbacks do not work properly with LwM2M observations because engine does not know when data is changed.
+    * Sensor samples are enabled by default for Thingy91 and disabled by default on nRF9160dk.
```

Comment:
```suggestion
    * Removed all read callbacks from sensor code because of an issue of read callbacks not working properly when used with LwM2M observations.
      This is due to the fact that the engine does not know when data is changed.
    * Sensor samples are now enabled by default for Thingy:91 and disabled by default on nRF9160 DK.
```